Simple-Projects/Simple_Projects/Pen&Paper/main.cpp

```diff
@@ -1,22 +1,18 @@
 #include <iostream>
-#include "Paper.h"
 #include "Pen.h"
 
 int main() {
-	Pen* kugel = new Pen();
-	Paper* papirus = new Paper();
+	Pen kugel;
+	Paper papyrus;
 
-	std::cout << "Paper: " << papirus->getSymbols() << "/" << papirus->getMaxSymbols() << std::endl;
-	std::cout << "Pen: " << kugel->getInkAmount() << "/" << kugel->getInkCapacity() << std::endl;
+	std::cout << "Paper: " << papyrus.getSymbols() << '/' << papyrus.getMaxSymbols() << std::endl;
+	std::cout << "Pen: " << kugel.getInkAmount() << '/' << kugel.getInkCapacity() << std::endl;
 
-	kugel->write(*papirus, "\nMY LATIN PROVERBS:\n\nIn Vino Veritas! \nVae Victus! \nDum spiro spero... \nVestis virum reddit... \nConsuetudinis magna vis est..");
-	papirus->show();
+	kugel.write(papyrus, "\nMY LATIN PROVERBS:\n\nIn Vino Veritas! \nVae Victis! \nDum spiro spero... \nVestis virum reddit... \nConsuetudinis magna vis est..");
```

Comment:
Isn't it Vae Victis? I know Kain used to say it with 'u' but IIRC it was an error?